### DISCUSSION TITLE
Spirit Level Colour Change - GoPro 11 Labs v2.30.70 March 2024 Release

### DISCUSSION THREAD

**Discussion by compoundvpn** [gopro/labs#812] [General]
After upgrading to v2.30.70 for GoPro Hero 11, the colour of the Spirit Level changed from the default white to yellow.

The Extensions page (https://gopro.github.io/labs/control/extensions/) states:

"LEVL=size - Add a spirit level to the rear LCD, where the size can be 1-9. e.g. oMLEVL=6. Color was added for HERO12. "

But I am unable to find the correct command to change the colour from yellow to red (or any other colour).

When I create the QRCode for *LEVL=9 the default is always yellow.  The Hello World example in this recent firmware update hints at which colour codes are available:
"\1H\2e\3l\4l\5o \6W\7o\8r\1l\2d"

1=Red
2=Green
3=Yellow
4=Indigo
5=Purple
6=Blue
7=White
8=Black

How do we change the default colour of the spirit level from yellow to any other colour?

**Comment by dnewman-gpsw** [comment]
LEVL has three colors with no controls.

White -  nothing is level
Yellow - the horizon is level
Green - the horizon and pitch is level
